src/utils.rs

```diff
@@ -15,6 +15,8 @@ pub(crate) enum NodeExitCode {
     Success = 0,
     /// Indicates the node version should be downgraded.
     ShouldDowngrade = 102,
+    /// Indicates the node launcher should attempt to run the shutdown script.
+    ShouldExitLauncher = 103,
```

Comment:
Since we're gonna expose new value in our public interface, please also update the `README.md` and `CHANGELOG.md` files.